src/compiler/crystal/crystal_path.cr

```diff
@@ -144,7 +144,6 @@ module Crystal
     end
 
     private def make_relative_unless_absolute(filename)
-      filename = "#{Dir.current}/#{filename}" unless filename.starts_with?('/')
       File.expand_path(filename)
     end
```

Comment:
This could easily be removed altogether.

Reply:
Care to send a PR?

Reply:
Sure, will do.

Reply:
#9095